### DISCUSSION TITLE
Building a community : Rallly wide open

### DISCUSSION THREAD

**Discussion by Dynnammo** [lukevella/rallly#575] [General]
:warning: This message follows the Crowdin discussion started [here](https://crowdin.com/project/rallly/discussions/9) :warning: 

# Context
Rallly is quite a big project. It has (when I write these lines):
-  almost **1800** stars on GitHub
- complete translations in a dozen of languages (see [here](https://crowdin.com/project/rallly)
- 2 major releases
- a blog
- a strong momentum : see evolution [here](https://ossinsight.io/analyze/lukevella/rallly#overview)

# The problem
I'm really concerned about the fact that @lukevella seems to handle 99% of the Rallly project (as a matter of fact, if I run a `git shortlog -s -n` I obtain these results : 
```
 387  Luke Vella
    13  Snyk bot
     4  Jonas Höbenreich
     4  dependabot[bot]
     3  Matthew Precious
     1  Dangersohn
     1  Girish Ramakrishnan
     1  Lars Haalck
     1  Manu
     1  Sam A
     1  Scott Wallace
     1  Trammell Hudson
```
TL;DR : the creator and maintainer of the project is responsible of **92%** of the commits of Rallly.
If I compare to similar OSS projects that have similar numbers, having such indicators while being alone is not a good thing.

As such stage, handling : 
- the code
- features askings
- translation process
- issues
- release process
- blog posting

requires much time, and it's the role of a community to help.

The AGPLv3 under which is licensed Rallly has the following motto : "Free as in freedom, not as in free beer", meaning users have the right to take advantage of the project, while having the responsability of taking care of it. 

# Solution**s**
As a member of multiple OSS communities, I think there's some actions that can be done in order to make Rallly a project that thrives with a community.
- **Discussions tools** : currently, the only way to discuss between members is with Github discussions. While it's practical since integrated with GitHub, it's not build for welcoming newbies. Tools like Discourse or Discord may help users of Rallly to talk to each other, helping them to discuss issues/features more informally
- **Contributing guide**: currently, the contribution part of the README mention only to reach out first the project manager and the translation process. It would be great to give some call-to-actions links like issue creation, fork link, link to GH discussions (or the future discussion tool if put in place)
- other ideas that might be relevant :smile:

**Comment by lukevella** [comment]
Hey @Dynnammo, thanks for taking the time to write this up. I think you've summed things up very well. The good thing is, there is already a [discord](https://discord.gg/uzg4ZcHbuM) which I've had help setting up from other members of the community. However, it's not very active and I haven't done much to change that. It would be great to get help with making that community more active.

I do have plans to write a contributing guide and to start creating issues that would be easy for new comers to pick up. On that note, I just spent some time labelling some of the [issues](https://github.com/lukevella/rallly/issues) and will try to create some more approachable ones for people to pick up. A contributing guide will follow.

Thanks again for opening this topic. I really appreciate the effort.
